### PR TITLE
fix api to fetch acn peer infos

### PR DESCRIPTION
**Pull request by vpiyush**
The wrong type was returned which didn't have any json tags either. so result was not displayed properly. Fixed the return type add test to make sure Json marshalling/unmarshalling is successful.

### CODE REVIEW

**Review by ScottRMalley — APPROVED** [review]
LGTM

File: params/generated/Oracle.go [hunk 1]
```diff
@@ -1,11 +1,8 @@
 package generated
 
-import (
-	"strings"
-
-	"github.com/autonity/autonity/accounts/abi"
-	"github.com/autonity/autonity/common"
-)
+import "strings"
```
Comment (yazzaoui) [inline params/generated/Oracle.go:3]: was that because of your local linting configuration?

**Review by yazzaoui — APPROVED** [review]
LGTM but please remove the oracle.go update from the commit before merging

File: params/generated/Oracle.go (hunk 1 above)
Comment (vpiyush) [reply params/generated/Oracle.go]: not because of that, but i couldn't figure why exactly that happened

**Pull request merged**